src/main/java/org/profit/telegrambot/MaksymenBot.java

```diff
@@ -0,0 +1,169 @@
+            if (update.getMessage().getText() != null) {
+                Message newMessage = update.getMessage();
+                String textMsg = newMessage.getText();
+                String chatId = String.valueOf(newMessage.getChatId());
+
+                if (textMsg.equals("/start")) {
+                    SendMessage sendMessage = new SendMessage();
+                    sendMessage.setText("Добридень, ми з України!\n" +
+                            "Введіть, будь ласка, своє ім'я та прізвище:");
+                    sendMessage.enableMarkdown(true);
+                    sendMessage.setChatId(chatId);
+                    try {
+                        execute(sendMessage);
+                    } catch (TelegramApiException e) {
+                        e.printStackTrace();
+                    }
+
+                } else if (textMsg.equals("Хліби")) {
+                    /*
+                    Для добавления одной кнопки:
+                    String textNewMsg = "Оберіть, який саме хліб Ви хочете замовити:";
+                    keyboard(chatId, textNewMsg, "Бородінський");
+                     */
+                    String textBread = "Оберіть, який саме хліб Ви хочете замовити:";
+                    sendInlineKeyboard(textBread, chatId, "Бородінський", "Василівський");
+                    // "Литовський", "Ризький", "Заріченський"
+                } else if (textMsg.equals("Торти")) {
+                    String textCake = "Оберіть, який саме торт Ви хочете замовити:";
+                    sendInlineKeyboard(textCake, chatId,  "Київський", "Мадонна");
+                    // "Сакура", "Йогуртовий", "Деметра"
+                } else if (textMsg.equals("Тістечка")) {
+                    String textCupcake = "Оберіть, яке саме тістечко Ви хочете замовити:";
+                    sendInlineKeyboard(textCupcake, chatId,  "Еклер", "Лимонне");
+                    // "Спартак",  "Корзинка", "Медове"
+                } else if (textMsg.equals("Круасани")) {
+                    String textPie = "Оберіть, який саме круасан Ви хочете замовити:";
+                    sendInlineKeyboard(textPie, chatId,  "З вишнею", "Зі згущеним молоком");
+                    // "З малиною", "З шоколадом", "Зі смородиною"
+                } else {
+                        String textNewMsg = textMsg + ", ми пропонуємо:";
+                        sendReplayKeyboard(textNewMsg, chatId);
+                    }
+                }
+            }
+        }
+
+
+
+        private void sendReplayKeyboard (String textNewMsg, String chatId){
+            SendMessage sendMessage = new SendMessage();
+            sendMessage.enableMarkdown(true);
+            sendMessage.setText(textNewMsg);
+            sendMessage.setChatId(chatId);
+            ReplyKeyboardMarkup keyboardMarkup =
+                    buttonService.setButtons(buttonService.createButtons(asList(
+                            "Хліби", "Торти", "Тістечка", "Круасани"
```

Comment:
🇺🇦 is just great

But most of the products have 🇬🇧 as a default or primary language.
Let us follow the trend and use English whenever it is possible.

As an additional task we could have i18n and l10n where you would add support for 🇺🇦